lago/virt.py

```diff
@@ -391,7 +393,7 @@ def _libvirt_xml(self):
             )
             net_xml.append(domain_xml)
         if 'dhcp' in self._spec:
-            IPV6_PREFIX = 'fd8f:1391:3a82:5e0d::'
+            IPV6_PREFIX = 'fd8f:1391:3a82:' + subnet + '::'
```

Comment:
@nvgoldin - it translates to IPv6 eventually, no worries. It's just few octets out of them, for example:
fd8f:1391:3a82:204::/64 - for 192.168.204 network.